src/sass/components/button.scss

```diff
@@ -168,10 +275,24 @@
   @include tetra-button(tertiary);
 }
 
+.btn.btn-default-alt {
```

Comment:
It is nested because for now to avoid an impact on dropdown buttons (since those are not designed right now).